src/els_document_highlight_provider.erl

```diff
@@ -45,20 +45,17 @@ find_highlights(Uri, #{ kind := Kind
                       }) when Kind =:= application;
                               Kind =:= implicit_fun;
                               Kind =:= function;
-                              Kind =:= variable;
```

Comment:
`variable` was removed since they are not indexed and included in the references table.

Reply:
At some point I would like to see highlights of local variables, according to the language scoping rules. But I guess that is not an indexing feature, it can come from the compiler for the specific module.

Reply:
I think it is possible to make this work through reference, which we could use also to go to the definition of a variable, but this would require some more complex analysis during parsing.

Reply:
I think it will add a lot of bulk to the index, and it might be simpler to just compute it as needed. It will always be in a local scope, and erlang functions tend to be quite small.  And no global variables.

Reply:
That makes a lot more sense 😄.